### PR TITLE
Turkish language

### PR DESCRIPTION
**Pull request by TaylanTatli**

### CODE REVIEW

File: translation/tr.po [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+"PO-Revision-Date: 2021-01-21 19:54+0300\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.4.1\n"
+"Last-Translator: \n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: tr\n"
+
```
Comment (phuhl) [inline translation/tr.po:16]:
```suggestion

```

Trying to trigger fixed CI

**Review by phuhl — CHANGES_REQUESTED** [review]

File: translation/tr.po [hunk 2]
```diff
@@ -13,7 +13,6 @@ msgstr ""
 "Last-Translator: \n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 "Language: tr\n"
-
 #: Main.hs:0
```
Comment (phuhl) [inline translation/tr.po:16]:
```suggestion

#: Main.hs:0
```

**Review by phuhl — CHANGES_REQUESTED** [review]

**Comment by phuhl** [comment]
Thanks for the contribution, merged it after fixing the CI that had some issue with not updating the apt-get cache.

**Comment by TaylanTatli** [comment]
> Thanks for the contribution, merged it after fixing the CI that had some issue with not updating the apt-get cache.

Thanks but it won't be installed since it's not in makefile. Same for `bn_BD` translation. Only `de` and `en` are installed.

**Comment by phuhl** [comment]
Uh, oh. True, would you submit a PR that fixes that?

**Comment by TaylanTatli** [comment]
> Uh, oh. True, would you submit a PR that fixes that?

Done https://github.com/phuhl/linux_notification_center/pull/143

**Pull request merged**